### PR TITLE
Fixed Xbox Controller on macOS reporting negative values for the stic…

### PR DESCRIPTION
**Pull request by jechter**
…ks when represented as dpad buttons.

### CODE REVIEW

File: Packages/com.unity.inputsystem/InputSystem/Plugins/XInput/XInputSupport.cs [hunk 1]
```diff
@@ -117,15 +117,15 @@ public static void Initialize()
...
+    { ""name"" : ""leftStick/down"", ""offset"" : 2, ""format"" : ""SHRT"", ""parameters"" : ""clamp,clampMin=0,clampMax=1,invert=false"" },
     { ""name"" : ""rightStick"", ""offset"" : 10, ""format"" : ""VC2S"" },
     { ""name"" : ""rightStick/x"", ""offset"" : 0, ""format"" : ""SHRT"", ""parameters"" : """" },
     { ""name"" : ""rightStick/left"", ""offset"" : 0, ""format"" : ""SHRT"", ""parameters"" : """" },
     { ""name"" : ""rightStick/right"", ""offset"" : 0, ""format"" : ""SHRT"", ""parameters"" : """" },
     { ""name"" : ""rightStick/y"", ""offset"" : 2, ""format"" : ""SHRT"", ""parameters"" : ""invert"" },
-    { ""name"" : ""rightStick/up"", ""offset"" : 2, ""format"" : ""SHRT"", ""parameters"" : ""clamp,clampMin=-1,clampMax=1,invert=true"" },
-    { ""name"" : ""rightStick/down"", ""offset"" : 2, ""format"" : ""SHRT"", ""parameters"" : ""clamp,clampMin=-1,clampMax=1,invert=false"" }
+    { ""name"" : ""rightStick/up"", ""offset"" : 2, ""format"" : ""SHRT"", ""parameters"" : ""clamp,clampMin=-1,clampMax=0,invert=true"" },
+    { ""name"" : ""rightStick/down"", ""offset"" : 2, ""format"" : ""SHRT"", ""parameters"" : ""clamp,clampMin=0,clampMax=1,invert=false"" }
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/InputSystem/Plugins/XInput/XInputSupport.cs:128]: Don't we have a test somewhere that's supposed to make sure this works?

**Comment by Rene-Damm** [comment]
Just realized that this probably conflicts with https://github.com/Unity-Technologies/InputSystem/pull/416. That PR had dropped off my radar :/

File: Packages/com.unity.inputsystem/InputSystem/Plugins/XInput/XInputSupport.cs (hunk 1 above)
Comment (jechter) [reply Packages/com.unity.inputsystem/InputSystem/Plugins/XInput/XInputSupport.cs:128]: Not for this specifically. But, looking at XInputTests, I think we can (and should) add it.

**Comment by jechter** [comment]
Yeah, this definitely conflicts with #416. So let's land that, and I'll re-fix it based on that.

**Comment by Rene-Damm** [comment]
@jechter #416 has landed.

**Comment by jechter** [comment]
Replaced by https://github.com/Unity-Technologies/InputSystem/pull/471

**Pull request closed without merging**